features/enter_preferences.feature

```diff
@@ -0,0 +1,50 @@
+Feature: participants can enter preferences
+
+As a participant,
+So that I can select which times I like and get the times I want,
+I want to be able to rank the times.
+
+Background: A project is set up and participants are invited to rank times
+    Given a registered user with the email "raichu@berkeley.edu" with username "raichu" exists
+    And a registered user with the username "raichu" has a project named "CS169 Sections"
+    And the project named "CS169 Sections" has the following participants:
+    | email                  |
+    | empoleon@berkeley.edu  |
+    | charizard@berkeley.edu |
+
+    And the project named "CS169 Sections" has the following times:
+    | datetime               |
+    | Dec 1 2019 10:00 AM    |
+    | Dec 1 2019 1:00 PM     |
+
+    And a participant to project "CS169 Sections" with email "empoleon@berkeley.edu" has a secret_id "empoleonsecretid"
+    And a participant to project "CS169 Sections" with email "charizard@berkeley.edu" has a secret_id "charizardsecretid"
+
+
+  Scenario: User enters their preferences for times
+    When I access the time ranking page for project "CS169 Sections" from email "empoleon@berkeley.edu" and secret_id "empoleonsecretid"
+    Then I should see "Please enter your preferences for these times."
+    And I should not see "Access denied"
+    When I choose "Cannot go" for time "Dec 1 2019 10:00 AM"
```

Comment:
This is really nice! Thanks for writing the tests and steps to make this possible, it's very flexible and easy to test.